src/Bridge/Doctrine/Orm/ItemDataProvider.php

```diff
@@ -154,24 +152,42 @@ private function getIdentifiers(string $resourceClass, $id)
      * @param ObjectManager $resourceManager
      * @param string        $resourceClass
      *
+     * @throws RuntimeException
+     *
      * @return \Doctrine\ORM\QueryBuilder
      */
     private function retrieveQueryBuilder(ObjectManager $resourceManager, string $resourceClass)
     {
         $repository = $resourceManager->getRepository($resourceClass);
 
-        if ($repository instanceof  EntityRepository) {
+        if ($repository instanceof EntityRepository) {
             return $repository->createQueryBuilder('o');
         }
 
         if ($resourceManager instanceof EntityManagerInterface) {
```

Comment:
I find this hack pretty ugly but maybe I missed something, it's been some time since I used Doctrine, would be happy to find a better solution